src/DataRenderer.tsx

```diff
@@ -25,6 +25,8 @@ export interface JsonRenderProps<T> {
   level: number;
   style: StyleProps;
   shouldExpandNode: (level: number, value: any, field?: string) => boolean;
+  onNodeExpanded?: (level: number, value: any, field?: string) => void;
```

Comment:
Not sure which way is better, but how about to combine these two callbacks into one?
```typescript
  onNodeExpandToogle?: (expanded: boolean, level: number, value: any, field?: string) => void;
```

It can simplify library code, e.g. we can remove parts like `if (shouldExpand) {`

Reply:
yeah I actually like that better, I did that at first and then changed it for some reason to this haha, will use the single callback instead